src/app/spaces/space.service.ts

```diff
@@ -206,7 +206,7 @@ export class SpaceService {
 
     if (!space.relationships['owned-by'] || !space.relationships['owned-by'].data) {
       space.relationalData.creator = null;
-      return;
+      return emptyObservable();
```

Comment:
RxJS has deprecated `emptyObservable` and now uses `EMPTY`

Reply:
Is it OK to return an empty observable or should I rather return `this.handleError('some message')`?

Reply:
If you want it to fail and report it then return an error.